### PR TITLE
pin-1825: Endpoint to return an eservice descriptor

### PR DESCRIPTION
**Pull request by galales**

### CODE REVIEW

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -628,17 +692,18 @@ components:
...
         - name
-        - version
-        - state
+        - category
+    Origin:
+      description: Attribute origin admitted
+      type: string
+      enum:
+        - IPA
+        - SDG
```
Comment (galales) [inline src/main/resources/interface-specification.yml:983]: this will need to be maintained manually

Comment (TonioGela) [reply src/main/resources/interface-specification.yml:983]: Exactly. Did I miss some decision about it? Why don't expose a mere string?

**Review by TonioGela — APPROVED** [review]

File: src/main/resources/interface-specification.yml (hunk 1 above)
Comment (galales) [reply src/main/resources/interface-specification.yml:983]: If I remember, it was intended also as "documentation" of the existing origins, but I'm not sure if it's worth.

@beetlecrunch what do you think?
(given that we are still in the test phase, we should be able to change this before October)

Comment (beetlecrunch) [reply src/main/resources/interface-specification.yml:983]: Let's keep the enum for now.

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**